scw/errors.go

```diff
@@ -4,6 +4,11 @@ import (
 	"fmt"
 )
 
+// SdkError is a base interface for all Scaleway SDK errors.
+type SdkError interface {
```

Comment:
Shouldn't we keep this interface private and provide a function `IsSCWError(err error) bool`?